_includes/binary.html

```diff
@@ -14,9 +14,9 @@
 <style>
     #binary			{width: 100%;}
     #decimal1		{font-family: Arial, Helvetica, sans-serif; float: left; font-size: 5vw; width: 21vw; margin: 2.7vw 0 0; float: left}
-    .column			{font-family: Arial, Helvetica, sans-serif; float: left; text-align: center; width: 54px}
+    .column			{font-family: Arial, Helvetica, sans-serif; float: left; text-align: center; width: fit-content}
     .column_heading	{font-size: 1.6vw; color: #666666}
-    .bit			{font-size: 5vw; background-color: #FFFFFF; color:#000000; border-radius: 1.3vw; margin: 0.25vw; border: 1px solid grey}
+    .bit			{font-size: 5vw; background-color: #FFFFFF; color:#000000; border-radius: 1.3vw; margin: 0.25vw; border: 1px solid grey; padding: 3px}
```

Comment:
Why have you added the padding? There's no need to add padding I think so. After, adding the padding the outer box disappears. You can check and update the PR again once. Thanks!

Reply:
@fluctix Since it was running fine for others and not on Deepak's system. He has reviewed and suggested the above changes. So you can go ahead and make the changes.